### DISCUSSION TITLE
Issue creation template improvements

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1472] [Ideas]
- make it clearer for people to see what commit hash they should share and where to find it
- in the Game Version field, make the latest game version as the one ''pre-filled in'' and in the description of the field
- explain how to identify (at least briefly) which modules are affected
- change `See ScriptHookV.dll to check.` to `See ScriptHookV.log to check.` in the Game Version field as one can't copy the version number directly from the DLL

**Comment by kagikn** [comment]
> change See ScriptHookV.dll to check. to See ScriptHookV.log to check. in the Game Version field as one can't copy the version number directly from the DLL

good catching indeed, will fix this

> make it clearer for people to see what commit hash they should share and where to find it

I think that's clear enough for most people when one report bugs using a nightly release, since each nightly release has a unique commit hash for easier reference?

> in the Game Version field, make the latest game version as the one ''pre-filled in'' and in the description of the field

Probably not worth it, because I see most people who create bug report fill in a game version in the correct format even without updating the field. You can't use scripts in issue templates so you can fetch some values programmatically, either.

> explain how to identify (at least briefly) which modules are affected

Do you have any idea why I wrote this following quote? 

> If you are not familiar with how to program and you are encountering some issues related to some SHVDN scripts such as compatibility issues due to game versions, please try to seek help from the authors and ask them to find out the causes.

Because without sufficient knowledge of how to program, one may not be able to identify what is exactly wrong with SHVDN. You seem to fail to avoid specifying the scripting dlls as affected modules in #1473, too. If one doesn't know how to program much and can't really identify what's wrong, the best bet would be asking those who know how to program good enough to identify issues and report issues for them who don't know how to program much.

**Reply by Michael21107** [reply]
> Because without sufficient knowledge of how to program, one may not be able to identify what is exactly wrong with SHVDN. You seem to fail to avoid specifying the scripting dlls as affected modules in #1473, too. If one doesn't know how to program much and can't really identify what's wrong, the best bet would be asking those who know how to program good enough to identify issues and report issues for them who don't know how to program much.

I think issues are there to be from people, regardless of their background and ability to identify what individual internal components of a program are in work. The quoted text makes me realize that an explanation of individual modules probably isn't feasible, but I feel like it's a no-brainer to allow the field to be left blank, at least while warning users of this (if possible).
